drivers/disk/Kconfig.sdmmc

```diff
@@ -2,7 +2,7 @@
 # Copyright (c) 2021 Nordic Semiconductor ASA
 # SPDX-License-Identifier: Apache-2.0
 
-DT_STM32_SDMMC_HAS_DMA := $(dt_nodelabel_has_prop,sdmmc,dmas)
+DT_STM32_SDMMC_HAS_DMA := $(dt_nodelabel_has_prop,sdmmc1,dmas)
```

Comment:
Tbh, I'm not actually quite sure how this used to work, but this would obviously not work for sdmmc2 (for instance available on stm32f765).
`dt_compat_any_has_prop` would solve the problem

Reply:
Thanks for the tip, switched to use `dt_compat_any_has_prop` instead.